pulumi/src/statefulService.ts

```diff
@@ -355,4 +356,8 @@ export async function deployStatefulService(
       { provider }
     )
   }
+
+  if(config.statefulService.backupCount && config.statefulService.backupSchedule){
+    deployStsBackupCron(asset, config.statefulService, namespace, provider);
```

Comment:
yeah we need to get formatting fixed up, i definitely botched things pulling out the shared pulumi code...
```suggestion
    deployStsBackupCron(asset, config.statefulService, namespace, provider)
```

Reply:
we should also only deploy backups only for sts in `unchained-dev`